mappings/net/minecraft/class_424.mapping

```diff
@@ -1,3 +1,5 @@
 CLASS net/minecraft/class_424
+	METHOD method_1005 loadDimension (Lnet/minecraft/class_425;)Lnet/minecraft/class_426;
+	METHOD method_1006 backupLevelData (Lnet/minecraft/class_427;Ljava/util/List;)V
```

Comment:
Should this be backupWorldData or no? I haven't checked the code on any of this yet.